guides/common/modules/ref_system-requirements.adoc

```diff
@@ -93,7 +93,19 @@ ifdef::foreman-el,katello,satellite[]
 .SELinux mode
 SELinux must be enabled, either in enforcing or permissive mode.
 Installation with disabled SELinux is not supported.
+endif::[]
+
+.Synchronized system clock
+The system clock on the base operating system where you are installing your {ProductName} must be synchronized across the network.
+If the system clock is not synchronized, SSL certificate verification might fail.
+ifdef::satellite[]
+For more information, see the following documents:
```

Comment:
I also rephrased this to emphasize that chronyd is just one of the services that can be used for timekeeping.